examples/RestEaseClientGeneratorConsoleApp/Examples/Pitane/IPitaneApi.cs

```diff
@@ -0,0 +1,5046 @@
+
+    /// <summary>
+    /// in case the TO demands a deposit before usage. Requesting and refunding should be done using the /payment/claim-extra-costs endpoint.
+    /// </summary>
+    public class ConditionDeposit : Condition
+    {
+        /// <summary>
+        /// This should be in the base unit as defined by the ISO 4217 currency code with the appropriate number of decimal places and omitting the currency symbol. e.g. if the price is in US Dollars the price would be 9.95. This is inclusive VAT
+        /// </summary>
+        public double Amount { get; set; }
+
+        public double AmountExVat { get; set; }
+
+        /// <summary>
+        /// ISO 4217 currency code
+        /// </summary>
+        public string CurrencyCode { get; set; }
+
+        /// <summary>
+        /// value added tax rate (percentage of amount)
+        /// </summary>
+        public double VatRate { get; set; }
+
+        /// <summary>
+        /// two-letter country codes according to ISO 3166-1
+        /// </summary>
+        public string VatCountryCode { get; set; }
+
+    }
+
+    /// <summary>
+    /// in case the TO demands a direct payment after usage.
+    /// </summary>
+    public class ConditionPayWhenFinished : Condition
+    {
+    }
+
+    public class ConditionPostponedCommit : Condition
+    {
+        public DateTime UltimateResponseTime { get; set; }
+    }
+
+    public class ConditionRequireBookingData : Condition
+    {
+        public string[] RequiredFields { get; set; }
+    }
+
+    /// <summary>
+    /// a return area. In the condition list there can be multiple return area's.
+    /// </summary>
+    public class ConditionReturnArea : Condition
+    {
+        /// <summary>
+        /// station to which the asset should be returned
+        /// </summary>
+        public string StationId { get; set; }
+
+        /// <summary>
+        /// area in which the asset should be returned as GeoJSON Polygon coordinates
+        /// </summary>
+        public ConditionReturnAreaReturnArea ReturnArea { get; set; }
+
+        /// <summary>
+        /// a lon, lat (WGS84, EPSG:4326)
+        /// </summary>
+        public Coordinates Coordinates { get; set; }
+
+        /// <summary>
+        /// the return hours of the facility (if different from operating-hours)
+        /// </summary>
+        public SystemHours[] ReturnHours { get; set; }
+    }
+
+    /// <summary>
+    /// area in which the asset should be returned as GeoJSON Polygon coordinates
+    /// </summary>
+    public class ConditionReturnAreaReturnArea
+    {
+        /// <summary>
+        /// geojson representation of a polygon. First and last point must be equal. See also https://geojson.org/geojson-spec.html#polygon and example https://geojson.org/geojson-spec.html#id4. The order should be lon, lat [[[lon1, lat1], [lon2,lat2], [lon3,lat3], [lon1,lat1]]], the first point should match the last point.
+        /// </summary>
+        public double[][][]  { get; set; }
```

Comment:
I noticed we have a name missing here, maybe returning an empty string isn't that good of an idea. Or is this related to something else?